From 7c50f4562a8a7d71250243b8fdf97d4279f5bada Mon Sep 17 00:00:00 2001
From: Zhu Yuqing <b48954@freescale.com>
Date: Thu, 12 Jun 2014 10:27:28 +0800
Subject: [PATCH 071/115] ENGR00303287 fix OTA: Last part is cut for OTA
 upgarde description

change new version display from "Have new release" to "Have a new release"

Signed-off-by: Zhu Yuqing <b48954@freescale.com>
---
 FSLOta/res/values/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/FSLOta/res/values/strings.xml b/FSLOta/res/values/strings.xml
index 176175e..5ac62cf 100644
--- a/FSLOta/res/values/strings.xml
+++ b/FSLOta/res/values/strings.xml
@@ -9,7 +9,7 @@
         <string name="error_server_no_package">Error: server don\'t have upgrade package</string>
         <string name="error_write_file">Error: write file error</string>
         <string name="already_up_to_date">\nYour software is up to date</string>
-        <string name="have_new">Have new release</string>
+        <string name="have_new">Have a new release</string>
         <string name="error_package_install_failed">package install failed</string>
         <string name="error_needs_wifi">Error: needs wifi network.</string>
         <string name="error_cannot_connect_server">Error: Can not connect to server</string>
-- 
1.8.0

